com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.hlsl

```diff
@@ -494,62 +494,61 @@ void LightLoop( float3 V, PositionInputs posInput, PreLightData preLightData, BS
 #endif
 
 #if SHADEROPTIONS_ENABLE_PROBE_VOLUMES == 1
-    bool uninitialized = IsUninitializedGI(builtinData.bakeDiffuseLighting);
-    builtinData.bakeDiffuseLighting = uninitialized ? float3(0.0, 0.0, 0.0) : builtinData.bakeDiffuseLighting;
-
+    bool uninitializedGI = IsUninitializedGI(builtinData.bakeDiffuseLighting);
     // If probe volume feature is enabled, this bit is enabled for all tiles to handle ambient probe fallback.
-    // No need to branch internally on _EnableProbeVolumes uniform.
-    if (featureFlags & LIGHTFEATUREFLAGS_PROBE_VOLUME)
+    // Even so, the bound resources might be invalid in some cases, so we still need to check on _EnableProbeVolumes.
+    bool apvEnabled = featureFlags & LIGHTFEATUREFLAGS_PROBE_VOLUME && _EnableProbeVolumes;
+    if (!apvEnabled)
     {
-#if !SHADEROPTIONS_PROBE_VOLUMES_ADDITIVE_BLENDING
-        if (uninitialized)
-#endif
+        builtinData.bakeDiffuseLighting = (uninitializedGI && !apvEnabled) ? float3(0.0, 0.0, 0.0) : builtinData.bakeDiffuseLighting;
```

Comment:
So we need to define something here. what disabling APV mean?
For mean disabling APV mean we fallback on the previous probe system (and not to black)
So disabling APV shouldn't require this code above. It should just be when we sample probe, we use lightprobe or put an uninitializedGI  mark depends if APV is enabled or not.

Reply:
nevermind , misread the code :P